src/js/pensions/containers/ConfirmationPage.jsx

```diff
@@ -54,8 +55,18 @@ class ConfirmationPage extends React.Component {
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong><br/>
+              <address className="schemaform-address-view">{regionalOffice.map((line, index) =>
+                <p key={index}>{line}</p>)}</address>
```

Comment:
Yeah, like I said, it's not a problem here--even most places we use it--but it's something I saw the other day that we should keep an eye out for just in case. At any rate, I'm guessing uniqueId would probably be overkill here. I thought about it, though.